README.md

```diff
@@ -2564,7 +2554,7 @@ It’s not exhaustive, as titles and details vary.
 - Venture Hacks, [How to make a cap table](http://venturehacks.com/articles/cap-table)
 - VentureBeat,
   [Beware the trappings of liquidation preference](http://venturebeat.com/2010/08/16/beware-the-trappings-of-liquidation-preference/)
-- Orrick,
+- 📥 Orrick,
```

Comment:
If we could move all immediately before the link it'd be best, to be consistent.

Reply:
Can be a separate PR.